x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/get_error_count_chart_preview.ts

```diff
@@ -66,7 +63,9 @@ export async function getTransactionErrorCountChartPreview({
     bool: {
       filter: [
         ...termFilterQuery,
-        ...getParsedFilterQuery(kqlFilter),
+        ...getParsedFilterQuery(
+          searchConfiguration ? JSON.parse(searchConfiguration) : undefined
```

Comment:
This should be parsed at the route level. 
```suggestion
          searchConfiguration
```

Reply:
updated in bee1742c5c900538fe1131d9c0c0b6de350c6dcc